### PR TITLE
Broken links removed

### PR DESCRIPTION
**Pull request by AimTune**
## Description

Broken links removed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,15 +160,11 @@ Flutter ile ilgili Türkçe makaleler kategorilerine göre ayrılmış halde bur
 ##### DSC Edirne
 - [Flutter vs React Native](https://youtu.be/BtxsOehgl_s) - Sercan Yusuf & Özge Aydın
 
+<!--
 ##### Developer MultiGroup
```
Comment (Adem68) [inline README.md:164]: Can you update this section from this playlist?

https://www.youtube.com/watch?v=lVVcrBnGzEA&list=PLZj2Q42tw-68zEBhpO_2mwX-1roiSvEHd

**Review by Adem68 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -34,21 +34,18 @@ Flutter öğrenmeye başlayan pek çok insan nereden başlayacağını veya nas
...
-- [Flutter öğrenmek istiyorum, nereden başlamalıyım](https://medium.com/flutter-t%C3%BCrkiye/flutter-%C3%B6%C4%9Frenmek-i%CC%87stiyorum-nas%C4%B1l-ba%C5%9Flamal%C4%B1y%C4%B1m-650427b18cfe) - Beyza Sunay Güler'in Pooja Baumik'in makalesinden çevirisi
 - [Flutter ile ilk uygulamanı yazmaya hazır mısın? (Part 1)](https://medium.com/flutter-t%C3%BCrkiye/flutter-ile-i%CC%87lk-uygulaman%C4%B1-yazmaya-haz%C4%B1r-m%C4%B1s%C4%B1n-part-1-f2833af78685) - Ajda Akter'in Flutter codelab çevirisi
 - [Flutter Nedir ?](https://medium.com/@halilozel1903/flutter-nedir-8a3cb33fdb9d) - Halil Özel
 - [Android Studio’ya Flutter Kurulumu](https://medium.com/@halilozel1903/android-studioya-flutter-kurulumu-73f7d4cefde) - Halil Özel
 - [Flutter’ın Mimarisi ve Dart’ın Flutter üzerindeki etkisi](https://medium.com/flutter-t%C3%BCrkiye/flutter%C4%B1n-mimarisi-ve-dart-%C4%B1n-flutter-%C3%BCzerindeki-etkisi-b9b652d0525) - Muhammed Salih Güler
 - [Flutter’ da pro düzeye ulaşmak için gerekli adımlar](https://medium.com/@hubders/flutter-da-pro-d%C3%BCzeye-ula%C5%9Fmak-i%C3%A7in-gerekli-ad%C4%B1mlar-faa6837a03de) - DersHub
-- [Flutter için kod örnekleri.] (onursahin.net/flutter) - Onur Şahin @imonursahin
```
Comment (Adem68) [inline README.md:51]: Why you deleted this?

Comment (Adem68) [reply README.md:51]: You can change to https://onursahin.net/flutter

Comment (AimTune) [reply README.md:51]: I searched but couldn't find it. I will add this.

File: README.md (hunk 1 above)
Comment (AimTune) [reply README.md:164]: I will do it.

**Comment by Adem68** [comment]
LGTM

**Review by Adem68 — APPROVED** [review]

**Pull request merged**